src/devTools.js

```diff
@@ -120,6 +123,25 @@ function liftReducer(reducer, initialState) {
       currentStateIndex = 0;
       timestamps = [liftedAction.timestamp];
       break;
+    case ActionTypes.EXPORT:
+      monitorState = Object.assign(
+        {},
+        monitorState,
+        { exportMode: !monitorState.exportMode }
+      );
+      break;
+    case ActionTypes.IMPORT:
+      (
+        {
```

Comment:
Please remove extra indentation and line break here.
This is fine:

```js
({
  stagedActions,
  ...
} = liftedAction.newState);
```

Let's also rename `newState` to `nextLiftedState`.